src/components/Tool/Home/ToolList.ts

```diff
@@ -12,104 +12,99 @@ import {
   mdiAccountGroup,
   mdiWrench,
 } from '@mdi/js';
-import i18n from 'i18next';
```

Comment:
Translating directly in this file was giving an error when trying to get them onto the navbar, they would always come up as undefined. It was even affecting the tools landing page. I changed it to the interpolation method but I'm not sure if that is the best way. The "reports" dropdown translates them as a dynamic key I think also

Reply:
Hmm... I wonder if there's a circular dependency or something... 

What you have won't translate anything... We could come back to it... Or we could just hardcode them in the menu instead of looping through them. I don't think we need the tools landing page anymore so we could just have them in one place.

Reply:
Hmm ok I'll just leave it for now incase we do want to keep the landing page for some reason, since its not really needed in the original app either but its still there